### PR TITLE
fixes #3186

### LINKED ISSUE #3186: Remove unnessary content for pom file
- [x] I have searched the [issues](https://github.com/apache/incubator-dubbo/issues) of this repository and believe that this is not a duplicate.
- [x] I have checked the [FAQ](https://github.com/apache/incubator-dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

Now dubbo requires java 8 or above, I notice there are still some configuration or profile for java6 (and hudson?)

Those content is not nessary any more, we need clean up it.

### PR DESCRIPTION
**Pull request by kezhenxu94**
## What is the purpose of the change

This patch fixes issue #3186 

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
         <cglib_version>2.2</cglib_version>
         <mockito_version>2.23.4</mockito_version>
         <!-- Build args -->
-        <argline>-server -Xms256m -Xmx512m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
+        <argline>-server -Xms256m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
```
Comment (ralf0131) [inline pom.xml]: Dubbo 2.7 runs on Java 8, in Java 8 we need Metaspace and MaxMetaspace.

Comment (ralf0131) [reply pom.xml]: Hi, this is not resolved yet.

We need  `-XX:MetaspaceSize=64m -XX:MaxMetaspaceSize=128m` here.

Comment (kezhenxu94) [reply pom.xml]: Thanks, if any further changes are required, please let me know

**Comment by kezhenxu94** [comment]
no responding, close it

**Comment by ralf0131** [comment]
Hi, 

I am sorry for the slow response on this pull request. Is this the reason you close the pr?
Would you please re-send the pull request?

**Comment by htynkn** [comment]
LGTM, but I have one question about profile `test`, looks like it's not necessary. any idea @ralf0131

File: pom.xml [hunk 2]
```diff
@@ -99,7 +99,7 @@
         <mockito_version>2.23.4</mockito_version>
         <!-- Build args -->
         <argline>-server -Xms256m -Xmx512m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
```
Comment (ralf0131) [inline pom.xml]: Please remove `-XX:PermSize=64m -XX:MaxPermSize=128m`, there is no need to keep it.

**Comment by ralf0131** [comment]
> LGTM, but I have one question about profile `test`, looks like it's not necessary. any idea @ralf0131

Couldn't figure out the purpose too, +1 to remove it.

**Comment by htynkn** [comment]
no activity from origin PR owner. I'm going to merge this change and polish it later

**Pull request merged**